js/src/command.js

```diff
@@ -11,9 +10,14 @@ export default class Command {
     this.functionName = functionName;
     this.args = args;
     this.kwargs = kwargs;
+    this.execFn = undefined;
   }
 
-  execute() {
-    tf[this.functionPath](...this.args);
+  executeRoutine() {
+    if (typeof this.execFn === 'undefined') {
+      throw new TypeError(NOT_TRANSLATED);
+    }
+
+    return this.execFn(...this.args);
```

Comment:
Hmm, what about kwargs?
E.g. `torch.nn.functional.softmax(tensor, dim=1)` (where dim is kwarg) becomes `tfjs.softmax(tensor, 1)` where dim is arg.

Reply:
Translation will modify `args` to place the kwargs in the right spot